cves/CVE-2017-16840.yml

```diff
@@ -127,7 +133,10 @@ lifetime:
           git log --stat abc..def -- your/file.c
```

Comment:
This is the only place Github will let me write it. Please write a note about the fix, as well as any extra information you can provide.